cmd/main.go

```diff
@@ -129,14 +129,29 @@ func main() {
 		os.Exit(1)
 	}
 
-	if err = (&controllers.PDBWatcherReconciler{
+	if err = (&controllers.PodScalingReconciler{
+		Client: mgr.GetClient(),
+		Scheme: mgr.GetScheme(),
+	}).SetupWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create controller", "controller", "Pod Scaler")
+		os.Exit(1)
+	}
+
+	if err = (&controllers.DeploymentPDBReconciler{
```

Comment:
should the deployment reconciler have its own file?

Reply:
not sure what this means